global_helpers/panther_audit.py

```diff
@@ -1,14 +1,18 @@
 import os
 import boto3
 
+from panther_base_helpers import FIPS_ENABLED, FIPS_SUFFIX
```

Comment:
This isn't working even though it's the same import as others in the same directory :')

Reply:
Globals can't import from other globals in general. There's a workaround, but I'd prefer we don't use it and just either copy the relevant bits over or figure something else out.

Reply:
The logic to calculate `FIPS_ENABLED` and `FIPS_SUFFIX` can be done in this individual file. However, I thought putting the logic in some centralized location (not necessarily `panther_base_helpers`) would be best as it is used a few times. Let me know what you think. 👍

Reply:
Update: put the logic in each place where needed